code/game/objects/items/oddities.dm

```diff
@@ -0,0 +1,211 @@
+	if(!claimed)
+		claimed = TRUE
+		name = "[user.name]'s [name]"
+		return TRUE
+	else
+		return FALSE
+
+
+//Oddities are separated into categories depending on their origin. They are meant to be used both in maints and derelicts, so this is important
+//This is done by subtypes, because this way even densiest code monkey will not able to misuse them
+//They are meant to be put in appropriate random spawners
+
+//Common - you can find those everywhere
+/obj/item/weapon/oddity/common/blueprint
+	name = "strange blueprint"
+	desc = "You can't figure out what exactly this is. This is probably don't even works."
+	icon_state = "blueprint"
+	oddity_stats = list(
+		STAT_COG = 5,
+		STAT_MEC = 7,
+	)
+
+/obj/item/weapon/oddity/common/coin
+	name = "strange coin"
+	desc = "It look more like collectible rather than actual curency. But you can't figure out the alloy it made from."
+	icon_state = "coin"
+	oddity_stats = list(
+		STAT_ROB = 5,
+		STAT_THG = 5,
+	)
+
+/obj/item/weapon/oddity/common/photo_landscape
+	name = "alien landscape photo"
+	desc = "There some ire about this planet."
+	icon_state = "photo_landscape"
+	oddity_stats = list(
+		STAT_COG = 5,
+		STAT_THG = 5,
+	)
+
+/obj/item/weapon/oddity/common/photo_coridor
+	name = "surreal maint photo"
+	desc = "Something wrong, yet so familiar about this place."
+	icon_state = "photo_corridor"
+	oddity_stats = list(
+		STAT_MEC = 5,
+		STAT_THG = 5,
+	)
+
+/obj/item/weapon/oddity/common/photo_eyes
+	name = "observer photo"
+	desc = "What the fuck is this even."
+	icon_state = "photo_corridor"
+	oddity_stats = list(
+		STAR_ROB = 6,
+		STAT_THG = 6,
+		STAT_VIG = 6,
+	)
+
+/obj/item/weapon/oddity/common/old_newspaper
+	name = "old newspaper"
+	desc = "It contains a report on some old and strange phenomenon. Maybe it's lies, maybe it's corporate experements gone wrong."
+	icon_state = "old_newspaper"
+	oddity_stats = list(
+		STAT_MEC = 4,
+		STAT_COG = 4,
+		STAT_BIO = 4,
+	)
+
+/obj/item/weapon/oddity/common/paper_crumpled
+	name = "turn-out page"
+	desc = "This is ALMOST makes sense."
+	icon_state = "paper_crumpled"
+	oddity_stats = list(
+		STAT_MEC = 6,
+		STAT_COG = 6,
+		STAT_BIO = 6,
+	)
+
+/obj/item/weapon/oddity/common/paper_omega
+	name = "collection of obscure reports"
+	desc = "Even the authors seem to be rather skeptical about their findings. Yet they are not connected to each other, but results are simular."
+	icon_state = "paper_omega"
+	oddity_stats = list(
+		STAT_MEC = 8,
+		STAT_COG = 8,
+		STAT_BIO = 8,
+	)
+
+/obj/item/weapon/oddity/common/book_eyes
+	name = "observer book"
+	desc = "This book details the information on some cyber creatures. Who did this, how this is even possible?"
+	icon_state = "book_eyes"
+	oddity_stats = list(
+		STAR_ROB = 9,
+		STAT_THG = 9,
+		STAT_VIG = 9,
+	)
+
+/obj/item/weapon/oddity/common/book_omega
+	name = "occult book"
+	desc = "That's mostly does not make any logical sense. However, those stories are at least interesting."
```

Comment:
"Most of the stories in this book seem to be the writing of mad men. But at least the stories are interesting."